modules/service/publish-service.js

```diff
@@ -22,7 +22,9 @@ class PublishService {
         assertion.id = this.validationService.calculateHash(assertion.metadataHash + assertion.metadata.dataHash);
         assertion.signature = this.validationService.sign(assertion.id);
 
-        keywords.push(assertion.metadata.type);
+        if (assertion.metadata.type !== 'default') {
```

Comment:
Let's exclude any type from the keyword, not only 'default'

Reply:
Yes, it is better approach